code-examples/quarkus/kafka-quickstart/.kubernetes/kubernetes.yml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: v1
```

Comment:
In general this whole file needs a lot of comments - I would expect to see one for most lines.

Reply:
Yes. Decorate generated templates can be tweaked by properties. Would do that

Reply:
When is this template generated?

Reply:
It sounds like it possibly shouldn't be checked in at all?

Reply:
In my mind there will be two guides. One dev where requirement is to have local dev, quay or dockerhub account and one run application locally. 

Second guide is about how to deploy to OpenShift and connect with Managed Kafka with Operator. That guide technically can use kubernetes template. I will improve it a little bit but that is really what dekorate generates (we can tweak things for it.